File: 01_Language/01_Functions/python/array_change_key_case.py
# coding: utf-8

def array_change_key_case(d, case="CASE_LOWER"):
    f = str.lower if (case is None or case == "CASE_LOWER") else str.upper
    return dict((f(k), v) for k, v in d.items())


if __name__ == '__main__':
    d = dict([("FirSt", 1), ("SecOnd", 4)])
    print(array_change_key_case(d, "CASE_UPPER"))

File: 01_Language/01_Functions/php/array_change_key_case.php
<?php
$input_array = array("FirSt" => 1, "SecOnd" => 4);
var_dump(array_change_key_case($input_array, CASE_LOWER));

//array(2) {
//  ["FIRST"]=>
//  int(1)
//  ["SECOND"]=>
//  int(4)
//}
